py/templates/_default.html

```diff
@@ -7,6 +7,7 @@
     <title>{% block title %}{% endblock %} - Abbey2</title>
     <link href="{{ url_for('static', filename = 'css/bs.css') }}" rel="stylesheet">
     <link href="{{ url_for('static', filename = 'css/common.css') }}" rel="stylesheet">
+    <link rel="shortcut icon" href="{{ url_for('static', filename='favicon.ico') }}">
   </head>
```

Comment:
small suggestion - it's great to have different changes (e.g. ID cleanups and adding favicon) in different PRs... In this way they could be accepted independently - and what is more important, if for some reason later we need hastily revert one of them, the other is not affected :)

(but for now I think it's ok so no additional efforts are needed)